219 scales parameter for facet_wrap in g_spaghettiplot (#201)
insightsengineering/goshawk: NEWS.md

### Enhancements

* Implemented `nestcolor` with slight refactoring to `g_lineplot`, `g_density_distribution_plot` and added `nestcolor` in examples without custom color manuals.

> * `g_spaghettiplot` has new `facet_scales` parameter that allow to free the scales for missing factor levels within a piece of the grid in the final plot.

Reviewer: ```suggestion
* `g_spaghettiplot` has a new parameter `facet_scales` that allows for releasing scales in plot facets.
```
